### PR TITLE
Yr/meeting paderborn

### PR DESCRIPTION
**Pull request by YueRen**
Added a rough schedule and updated participants list

### CODE REVIEW

**Comment by YueRen** [comment]
Oops, just noticed that participants list is out of order, one moment.

**Comment by YueRen** [comment]
Nevermind, the names are in the correct order in the list (sorted by last name). Not sure what makes the website sort them by first name.

File: meetings/2023-09/participants.md [hunk 1]
```diff
@@ -10,11 +10,13 @@ people:
     - {name: Sebastian Degen, affiliation: Bielefeld University}
     - {name: Antony Della Vella, affiliation: TU Berlin}
     - {name: Claus Fieker, affiliation: RPTU Kaiserslautern-Landau}
+    - {name: Tommy Hofmann, affiliation: Siegen University}
```
Comment (thofma) [inline meetings/2023-09/participants.md]:
```suggestion
    - {name: Tommy Hofmann, affiliation: University of Siegen}
```

**Review by YueRen — COMMENTED** [review]
Looks good.  I'd approve it, but Github won't let me.

**Pull request merged**